examples/openmote_adc/main.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+#include "xtimer.h"
+#include "periph/adc.h"
+
+#define RES             ADC_RES_12BIT
+#define DELAY           (100 * 1000U)
+
+int main(void)
+{
+    uint32_t last = xtimer_now();
+    int16_t sample = 0;
+
+    adc_init(AD4_PIN);
+
+    while(1){
+
+        sample = adc_sample(AD4_PIN, RES);
+
+        if (sample < 0) {
```

Comment:
If you are using differential channels it will not be necessarily a positive value